Client/Frontend/Browser/TabDisplayManager.swift

```diff
@@ -205,6 +205,8 @@ class TabDisplayManager: NSObject {
             return
         }
         tabManager.removeTabAndUpdateSelectedIndex(tab)
+        
+        UnifiedTelemetry.recordEvent(category: .action, method: .close, object: .tab)
```

Comment:
There are a few other ways to close a tab, do you only want to capture this one?

Reply:
This takes care of swiping it closed and clicking the `x`, right? The other way would be holding the tab view button and selecting Close (which isn't captured right now, I will check if it should be). Are there others?

Reply:
Maybe it's better to this to `TabManager.removeTabAndUpdateSelectedIndex` which should take care of every case?

Reply:
Maybe simpler there. Could be inflated a tiny percentage (as tabs can be closed without user interaction), but that should be ok.